mediator/test/e2e-test.sh

```diff
@@ -6,6 +6,11 @@ MEDIATORDIR="${BASEDIR}/mediator"
 export NODE_ENV=integration
 export NODE_TLS_REJECT_UNAUTHORIZED=0
 
+export OPENMRS_HOST=openmrs
+export OPENMRS_USERNAME=admin
+export OPENMRS_PASSWORD=Admin123
+
+# Cleanup from last test, in case of interruptions
 retry_startup() {
   max_attempts=5
   count=0
```

Comment:
In `L17` I introduced a bug. I think the correct command would be `./startup.sh up-test` instead of `./startup.sh init`

Reply:
fixed with [59bf494](https://github.com/medic/cht-interoperability/pull/115/commits/59bf49435580d83a588dedc11ed6cfe965c30408)